recipes/fiji/20141125/meta.yaml

```diff
@@ -0,0 +1,24 @@
+about:
+    home: http://fiji.sc/
+    license: GNU General Public License
+    summary: Fiji is an image processing package—a "batteries-included" distribution of ImageJ, bundling a lot of plugins which facilitate scientific image analysis.
+package:
+    name: fiji
+    version: 20141125
+build:
+  number: 1
```

Comment:
I believe it is "mostly" working on OS X. :)  There is just a problem in the permissions / linking step in my environment.  I'll paste the loge here in case someone see it and knows if there's a quick fix.

```
$ conda build -c bioconda /Users/gvk/work/git_workspace/bioconda-recipes/recipes/fiji/20141125
Removing old build environment
Removing old work directory
BUILD START: fiji-20141125-1
Removing old work directory
Source cache directory is: /Users/gvk/miniconda2/conda-bld/src_cache
Found source in cache: fiji-macosx-20141125.dmg
Extracting download
Warning: Unrecognized source format. Source file will be copied to the SRC_DIR
Package: fiji-20141125-1
source tree in: /Users/gvk/miniconda2/conda-bld/work
+ set -eu -o pipefail
++ uname
+ unamestr=Darwin
+ '[' Darwin == Linux ']'
+ '[' Darwin == Darwin ']'
+ hdiutil attach fiji-macosx-20141125.dmg
Checksumming Driver Descriptor Map (DDM : 0)…
     Driver Descriptor Map (DDM : 0): verified   CRC32 $4A050171
Checksumming Apple (Apple_partition_map : 1)…
     Apple (Apple_partition_map : 1): verified   CRC32 $741825FE
Checksumming disk image (Apple_HFS : 2)…
..............................................................................
          disk image (Apple_HFS : 2): verified   CRC32 $D5315451
Checksumming  (Apple_Free : 3)…
                    (Apple_Free : 3): verified   CRC32 $00000000
verified   CRC32 $E920C8E2
/dev/disk2          	Apple_partition_scheme         	
/dev/disk2s1        	Apple_partition_map            	
/dev/disk2s2        	Apple_HFS                      	/Volumes/Fiji
+ cp -R /Volumes/Fiji/Fiji.app/ /Users/gvk/miniconda2/envs/_build
+ hdiutil detach /Volumes/Fiji
"disk2" unmounted.
"disk2" ejected.
++ sw_vers -productVersion
++ tr -d .
+ [[ 10114 -ge 1010 ]]
+ cp /Users/gvk/work/git_workspace/bioconda-recipes/recipes/fiji/20141125/ImageJ-macosx /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-macosx
+ rm /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-tiger
+ outdir=/Users/gvk/miniconda2/envs/_build/share/fiji-20141125-1
+ mkdir -p /Users/gvk/miniconda2/envs/_build/share/fiji-20141125-1
+ mkdir -p /Users/gvk/miniconda2/envs/_build/bin
+ cp /Users/gvk/work/git_workspace/bioconda-recipes/recipes/fiji/20141125/fiji.sh /Users/gvk/miniconda2/envs/_build/share/fiji-20141125-1/fiji
+ ln -s /Users/gvk/miniconda2/envs/_build/share/fiji-20141125-1/fiji /Users/gvk/miniconda2/envs/_build/bin
+ chmod 0755 /Users/gvk/miniconda2/envs/_build/bin/fiji
number of files: 392
Fixing permissions
Fixing linking of /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation in /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-macosx
Fixing linking of /System/Library/Frameworks/ApplicationServices.framework/Versions/A/ApplicationServices in /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-macosx
Fixing linking of /usr/lib/libSystem.B.dylib in /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-macosx
Fixing linking of /usr/lib/libgcc_s.1.dylib in /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-macosx
Fixing linking of /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices in /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-macosx
install_name_tool -add_rpath @loader_path/../../lib/ /Users/gvk/miniconda2/envs/_build/Contents/MacOS/ImageJ-macosx

Fixing linking of libgluegen-rt.dylib in /Users/gvk/miniconda2/envs/_build/java/macosx-java3d/Home/lib/ext/libgluegen-rt.jnilib
Error: Could not find libgluegen-rt.dylib
```